fickling/pickle.py

```diff
@@ -417,7 +445,10 @@ def has_non_setstate_call(self) -> bool:
     @property
     def is_likely_safe(self) -> bool:
         # `self.has_call` is probably safe as long as `not self.has_import`
-        return not self.has_import and not self.has_non_setstate_call
+        if self._vetted_dependencies is None or self._vetted_calls is None:
+            return not self.has_import and not self.has_non_setstate_call
+        else:
+            return not self.has_unvetted_import and not self.has_unvetted_calls
```

Comment:
shouldn't `is_likely_safe` be the `and` of all 4 conditions when applicable?

Reply:
`self.has_import` and `self.has_unvetted_import` as well as `self.has_non_setstate_call` and `self.has_unvetted_calls` would clash, given that the latter is more permissive than the former.